runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttOptionsConfigAdapter.java

```diff
@@ -110,7 +111,7 @@ public JsonObject adaptToJson(
         {
             JsonArrayBuilder versions = Json.createArrayBuilder();
             mqttOptions.versions
-                .forEach(versions::add);
+                .forEach(v -> versions.add(v == 4 ? MQTT_VERSION_4 : MQTT_VERSION_5));
```

Comment:
With the `MqttVersion` enum, you can keep the `versions::add`.

Reply:
We can simplify this, but not all the way to versions::add, since it's an unambiguous call (add(String), add(JsonValue))